gem/lib/earlgrey/configure_earlgrey.rb

```diff
@@ -66,6 +66,7 @@ def puts_yellow(string)
     def set_defaults(installer, project_name, test_target_name, scheme_file, opts = {})
       @swift = opts.fetch(:swift, false)
       @carthage = opts.fetch(:carthage, false)
+      @swift_version = opts.fetch(:swift_version, false)
```

Comment:
False is not a valid default value for `:swift_version`. `:swift` and `:carthage` are booleans which is why they default to false. `:swift_version` should default to a valid version of Swift.